tests/unit/streamalert/alert_processor/outputs/test_output_base.py

```diff
@@ -168,7 +168,7 @@ def test_check_http_response(self, mock_response):
     @mock_kms
     def test_load_creds(self):
         """OutputDispatcher - Load Credentials"""
-        param_name = '/{}_streamalert_secrets/{}'.format(
+        param_name = '/{}/streamalert/outputs/{}'.format(
             PREFIX, get_formatted_output_credentials_name('test_service', self._descriptor)
```

Comment:
I know this is annoying nit - but I don't like when other methods are called unnecessarily in a function that is testing something else that is very specific (`_load_creds` is being tested, `get_formatted_output_credentials_name` is called).. 

IMO this leads to confusion as to what the value of `param_name` actually is here and I'd personally rather hard-code the `param_name` value to what we expect it to be (otherwise I can't really tell what is being "used" for this value here..)

BUT.. maybe the actual problem here is that there aren't unit tests for `get_parameter_prefix` or `get_parameter_name` .. otherwise your above change would have broken other unit tests.

Reply:
Ok, I will refactor this to be clearer

Reply:
sorry 😑 😢